src/components/diff/PlainDiffComponent.tsx

```diff
@@ -6,6 +6,36 @@ import * as React from "react";
 import ReactResizeDetector from 'react-resize-detector';
 import { PullRequestCommentThreadModel, PullRequestFileModel, PullRequestPlainDiffCommentThreadModel } from "../../models";
 
+import * as monacoCSS from 'file-loader!../../../lib/JUPYTERLAB_FILE_LOADER_jupyterlab-pullrequests-css.worker.bundle.js';
```

Comment:
It's hard to tell which part of this came from `jupyterlab-monaco` and which part we've added. Could you add clear comments around the parts that have been ported over?